mirror_network.go

```diff
@@ -5,16 +5,18 @@ import (
 )
 
 type _MirrorNetwork struct {
-	networkNodes map[string]*_MirrorNode
-	network      []string
-	index        uint
+	networkNodes      map[*ManagedNodeAddress]*_MirrorNode
+	network           []*ManagedNodeAddress
+	index             uint
```

Comment:
This was removed from the Java SDK because this functionality is already achieved through the sorting of nodes.